app/packages/state/src/recoil/atoms.ts

```diff
@@ -410,3 +410,8 @@ export const sessionColorScheme = atom<ColorSchemeSetting>({
     fields: [],
   },
 });
+
+export const isUsingSessionColorScheme = atom<boolean>({
+  key: "isUsingSessionColorScheme",
+  default: false,
```

Comment:
Is this something we'd want to persist in browser storage?

Reply:
No, this is an indicator to determine whether to apply `dataset.appConfig.colorScheme` in loading. (when isUsingSessionColorScheme is false, apply dataset setting). It's set when we load the session state in `useStateUpdate`.